core/pom.xml

```diff
@@ -52,6 +52,10 @@
 			<artifactId>org.eclipse.jgit</artifactId>
 			<version>6.7.0.202309050840-r</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-model</artifactId>
+		</dependency>
```

Comment:
It's just to group dependencies

Reply:
I have and I do like the `pom` section but for the rest, there is already checkstyle and the `.editorconfig` file.